src/forms/components/inputs/multi-select/ohri-multi-select.component.tsx

```diff
@@ -104,8 +104,7 @@ export const OHRIMultiSelect: React.FC<OHRIFormFieldProps> = ({ question, onChan
             invalidText={errors[0]?.errMessage}
           />
         </div>
-        <div className={styles.formField}>
-          <OHRILabel value={'Selected Items'} />
+        <div className={styles.formField} style={{ marginTop: '.1%' }}>
```

Comment:
@pirupius am not an expert on this but I think this `.1%` is not a good practice, right? Instead here, I think @ODORA0 should use `xx rem` and follow the designs. Please keep an eye on this.

Reply:
@ebambo i had a similar concern so am working with him to resolve this

Reply:
@eudson @pirupius Changed to a more responsive rem

Reply:
Thank you guys